### DISCUSSION TITLE
Mappa termica

### DISCUSSION THREAD

**Discussion by zanac** [indomus/forum#2536] [GENERALE - Discussioni generiche]
Che dite, vi piace? Realizzato sbattendomi un po'...
![image](https://github.com/indomus/forum/assets/21194919/66432659-d6e2-4ad8-97ce-ccc4e74e64a6)

**Comment by angelopampalone** [comment]
Wow che bella card, puoi dare maggiori dettagli?

**Comment by zanac** [comment]
Certamente, la realizzazione è molto naif, se vuoi condivido ma è davvero una cosa vergognosa...

come card ho usato "custom:html-template-card" e dentro inserito in html una tabella dove staticamente su ogni cella punto al sensore virtuale. I sensori virtuali li ho creati usando ha-average... ve ne sono quindi ben 84, ogni uno calcola la media della temperatura su due ore... i colori ho fatto dei sensori template nei quali ho implementato il codice che calcola il gradiente RGB in base al valore di temperatura (quindi abbiamo 84 sensori gradiente). Ovviamente le celle della table html usano quindi anche il sensore gradiente per colorare la cella.
